client/src/pages/resources/[id]/index.js

```diff
@@ -1,74 +1,81 @@
 import React from 'react'
 import { Box, Heading, Anchor, Text, Button, Tabs, Tab } from 'grommet'
 import Link from 'next/link'
-import { fakeSearchMaterialsResponse } from '../../../helpers/testData'
+import api from '../../../api'
 import { ResourceDetails } from '../../../components/resources'
 import DetailsTable from '../../../components/DetailsTable'
 import Organism from '../../../images/organism.svg'
 import ResourceType from '../../../images/resource-type.svg'
 
-const ResourceDetailsPage = ({ resource }) => (
-  <>
-    <Box
-      direction="row"
-      justify="between"
-      align="center"
-      margin={{ bottom: 'large' }}
-    >
-      <Box pad="large">
-        <Heading level="5" margin={{ top: '0', bottom: 'large' }}>
-          <Link href="/resources/[id]" as={`/resources/${resource.id}`}>
-            <Anchor label={resource.title} />
-          </Link>
-        </Heading>
+const ResourceDetailsPage = ({ resource }) => {
+  if (!resource)
+    return (
+      <Text>This resource is not available. If this is an error... whoa</Text>
```

Comment:
The error message makes me think we won't actually intend to display this to the user ever, but it looks like we will if the API fails to respond correctly when the resource's deets are requested. Should this just say "This resource is not currently available. Please try again soon."?

Reply:
I updated the message, you are right that this will never get show and actually we will need to set up an error boundary so we can capture unrecoverable errors too. This one would probably a 404 style page if someone deletes a resource.